packages/javascript/bh-shared-ui/src/components/ExploreTable/useExploreTableRowsAndColumns.tsx

```diff
@@ -14,44 +14,35 @@ type DataTableProps = React.ComponentProps<typeof DataTable>;
 
 const filterKeys: (keyof MungedTableRowWithId)[] = ['displayname', 'objectid'];
 
-type UseExploreTableRowsAndColumnsProps = Pick<
-    ExploreTableProps,
-    'onKebabMenuClick' | 'allColumnKeys' | 'selectedColumns' | 'data'
-> & { searchInput: string };
+type UseExploreTableRowsAndColumnsProps = Pick<ExploreTableProps, 'onKebabMenuClick' | 'selectedColumns'> & {
+    searchInput: string;
+};
 
 const useExploreTableRowsAndColumns = ({
     onKebabMenuClick,
     searchInput,
-    allColumnKeys,
     selectedColumns,
-    data,
 }: UseExploreTableRowsAndColumnsProps) => {
     const [sortBy, setSortBy] = useState<keyof MungedTableRowWithId>();
     const [sortOrder, setSortOrder] = useState<'asc' | 'desc'>();
 
+    const { data: graphData } = useExploreGraph();
+    const exploreTableData = getExploreTableData(graphData);
+
     const rows = useMemo(
         () =>
-            (data &&
-                Object.entries(data).reduce((acc: MungedTableRowWithId[], curr) => {
-                    const [key, value] = curr;
-
-                    const valueAsPotentialEdge = value as StyledGraphEdge;
-
-                    if (!!valueAsPotentialEdge.id1 || !!valueAsPotentialEdge.id2) {
-                        return acc;
-                    }
-
-                    const nextRow = Object.assign({}, value.data);
-
-                    nextRow.id = key;
-                    nextRow.displayname = value?.label?.text;
-
-                    acc.push(nextRow as MungedTableRowWithId);
-
-                    return acc;
-                }, [])) ||
-            [],
-        [data]
+            ((exploreTableData?.nodes &&
```

Comment:
another benefit of pulling the GraphResponse directly is that we dont have to filter out the edges